From: Steve Lhomme <robux4@ycbcr.xyz>
Date: Wed, 4 Jan 2023 14:51:51 +0100
Subject: archive: fix the type used for libarchive_skip_cb

It is not using off_t.

(cherry picked from commit 5da9f85b186a69e5b64d83470b32b7fd19dbf943)
---
 modules/stream_extractor/archive.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/stream_extractor/archive.c b/modules/stream_extractor/archive.c
index 75bbe01..c69c9f2 100644
--- a/modules/stream_extractor/archive.c
+++ b/modules/stream_extractor/archive.c
@@ -131,7 +131,7 @@ static int libarchive_jump_cb( libarchive_t* p_arc, void* p_obj_current,
 
 
 static la_int64_t libarchive_skip_cb( libarchive_t* p_arc, void* p_obj,
-  off_t i_request )
+                                      la_int64_t i_request )
 {
     VLC_UNUSED( p_arc );
 
